apps/web/app/[locale]/timesheet/[memberId]/components/CalendarView.tsx

```
@@ -142,3 +148,95 @@ const CalendarDataView = ({ data, t }: { data?: GroupedTimesheet[], t: Translati
+                                                    className={cn(
+                                                        'shadow-[0px_0px_15px_0px_#e2e8f0] dark:shadow-transparent'
+                                                    )}
+                                                    taskTitleClassName={cn(
+                                                        'text-sm !text-ellipsis !overflow-hidden !truncate !text-[#293241] dark:!text-white '
+                                                    )}
+                                                    showSize={true}
+                                                    dash
+                                                    taskNumberClassName="text-sm"
+                                                />
+                                                <div>
+                                                    <span className="flex-1">{task.project && task.project.name}</span>
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)